### PR TITLE
Initial contribution of Blukii Smart Beacon support for Bluetooth

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
This is the first specific-device-add-on for the Bluetooth binding.
It brings support for [Blukii Smart Beacon](http://www.blukii.com/en/produkte/smartbeacons.html)s.

The build is still expected to fail as this PR requires #5208 to be merged first.

File: extensions/binding/org.eclipse.smarthome.binding.bluetooth.blukii/src/main/java/org/eclipse/smarthome/binding/bluetooth/blukii/BlukiiBindingConstants.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import org.eclipse.smarthome.binding.bluetooth.BluetoothBindingConstants;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link BlukiiBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Kai Kreuzer - Initial contribution
+ */
+public class BlukiiBindingConstants {
```
Comment (amitjoy) [inline extensions/binding/org.eclipse.smarthome.binding.bluetooth.blukii/src/main/java/org/eclipse/smarthome/binding/bluetooth/blukii/BlukiiBindingConstants.java]: Just a Geschmacksache:

```java
private BlukiiBindingConstants() {
     throw new IllegalAccessError("Cannot be instantiated");
}
```

File: extensions/binding/org.eclipse.smarthome.binding.bluetooth.blukii/src/main/java/org/eclipse/smarthome/binding/bluetooth/blukii/BlukiiBindingConstants.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_BEACON = new ThingTypeUID(BluetoothBindingConstants.BINDING_ID,
+            "blukii_beacon");
+
+    public final static String BLUKII_PREFIX = "blukii ";
```
Comment (amitjoy) [inline extensions/binding/org.eclipse.smarthome.binding.bluetooth.blukii/src/main/java/org/eclipse/smarthome/binding/bluetooth/blukii/BlukiiBindingConstants.java]: We could reorder the modifiers to comply with the Java Language Specification.

public final static ==> public static final

Comment (amitjoy) [reply extensions/binding/org.eclipse.smarthome.binding.bluetooth.blukii/src/main/java/org/eclipse/smarthome/binding/bluetooth/blukii/BlukiiBindingConstants.java]: applies to other constants as well.

**Comment by sjka** [comment]
It's always easy to review what _is_ there but I just realized I missed something which is _not_ here: Shouldn't this binding have a `README.md`?

**Pull request merged**